### PR TITLE
Latest files from Crowdin

### PR DESCRIPTION
**Pull request by zapbot**
Autogenerated PR from zapbot

### CODE REVIEW

File: src/lang/Messages_nl_NL.properties [hunk 1]
```diff
@@ -38,7 +38,11 @@ alert.source.tool=Greedschap
 
 alerts.name=Meldings Extensie
 alerts.desc=Staat u toe waarschuwingen te bekijken en te beheren
-alerts.label.defaultMessage=Volledige details van elke geselecteerde waarschuwing zal hier worden weergegeven. \n\nU kunt handmatig waarschuwingen toevoegen door met de rechtermuisknop te klikken op de relevante regel in de geschiedenis en ''Waarschuwing toevoegen'' te selecteren.\n\nU kunt ook bestaande waarschuwingen aanpassen door er dubbel op te klikken.
+alerts.label.defaultMessage=Volledige details van elke geselecteerde waarschuwing zal hier worden weergegeven. \n\
+\n\
```
Comment (kingthorin) [inline src/lang/Messages_nl_NL.properties]: What's with the trailing slashes?

Comment (thc202) [reply src/lang/Messages_nl_NL.properties]: Seems to be an issue in Crowdin.

Comment (kingthorin) [reply src/lang/Messages_nl_NL.properties]: Thanks!

**Comment by thc202** [comment]
@zapbot should not be raising new Crowdin PRs for the time being (it will also give us time to update Crowdin for zap-extensions changes).

**Pull request closed without merging**